doc/source/ray-core/api/cli.rst

```diff
@@ -1,5 +1,5 @@
-Ray Core CLI
-============
+Core CLI
```

Comment:
btw, not a big deal, but explicitly making headings less expressive is basically the opposite of the "doc search sanity check" I'm proposing. In Ray, we have so many common words like "core", "air", or "cluster" that are hard to differentiate. Nobody outside this org knows what "Core CLI" is.